src/code/db_camera.c

```diff
@@ -1631,9 +1631,9 @@ s32 DbCamera_LoadCallback(char* c) {
 s32 DbCamera_SaveCallback(char* c) {
     s32 pad[2];
     s32 ret;
-    u32 freeSize;
-    s32 off;
-    s32 size;
+    size_t freeSize;
+    ptrdiff_t off;
+    ptrdiff_t size;
     s32 i;
 
     ret = Mempak_GetFileSize(2, *c);
```

Comment:
There are several changes missing in this function, at least, and likely elsewhere in this file.